site2/docs/reference-metrics.md

```diff
@@ -260,6 +260,26 @@ All the managedLedger metrics are labelled with the following labels:
 | pulsar_ml_ReadEntriesSucceeded | Gauge | The number of readEntries requests that succeeded |
 | pulsar_ml_StoredMessagesSize | Gauge | The total size of the messages in active ledgers (accounting for the multiple copies stored) |
 
+### Managed cursor acknowledgment state
+
+The acknowledgment state is persistent to the ledger first. When the acknowledgment state fails to be persistent to the ledger, they are persistent to ZooKeeper. To track the stats of acknowledgment, you can configure the metrics for the managed cursor.
```

Comment:
`When persisting to the ledger fails, they will be persisted to ZooKeeper` 
may be better

Reply:
@hangc0276 thanks for your suggestion, but `When persisting to the ledger fails, they will be persisted to ZooKeeper` is grammatically inaccurate because it lacks a subject.

The original sentence is grammatically correct and it clearly indicates the subject (`acknowledgment state`).

So could you help approve this PR? Then this PR can be merged, thanks.

Reply:
OK

Reply:
@hangc0276 thanks!